### ISSUE TITLE
CC support?

### ISSUE THREAD

**Issue by nutinshell** [yuliskov/SmartYouTubeTV#263]
Plan to support subtitles/Captions display?

**Comment by f-o** [comment]
As far as I remember, subtitles were a feature a couple of updates ago.
You could enable and change the subtitles directly on the playback interface, without going into settings. 

I am too bummed this feature has been removed, but maybe there's a reason.

**Comment by nutinshell** [comment]
I hope it will come back much sooner :)

**Comment by f-o** [comment]
Hey again,
There was a new update released today, and I immidiately thought about this issue. 
When it had finished installing and I loaded a video, I was still bummed that the CC button was missing.
That's when I started messing around, and belive it or not - It's been there all along!

When on the playback interface, select "QUALITY".
You'll get a new row, with many of the missing features.
Subtitles is there as well!

Not sure why it has been hidden away under a quality label, but I don't care since it's there and works like a charm.
